pkg/container/runtime/types.go

```diff
@@ -186,6 +186,8 @@ const (
 	TypeDocker Type = "docker"
 	// TypeKubernetes represents the Kubernetes runtime
 	TypeKubernetes Type = "kubernetes"
+	// TypeColima represents the Colima runtime
```

Comment:
This should not be necessary - it looks like you are using the Docker runtime to connect to Colima. This enum is used for different implementations of our runtime interface.

Reply:
Maybe I should remove the Colima check in the docker runtime codepath:
https://github.com/stacklok/toolhive/pull/1332/files#diff-d58624f599f4b93a4c84d22c3d24888c6a2c57855eb451ccc72815d5cda9667bR65-R73